deployment/terraform/templates/pulsar.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
```

Comment:
this empty script is intentionally present?

Reply:
@rdhabalia this is still wip :) I am placing a empty script and updating the init script one by one